src/convertToObject.js

```diff
@@ -12,7 +12,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleMap = sourceString.split('\n').map(el => el.trim()).filter(el =>
+    el !== '' && el !== ';').map(el => {
+    return el.split(':').map(item => item.trim());
+  });
```

Comment:
```suggestion
    el !== '' && el !== ';').map(el => (
      el.split(':').map(item => item.trim());
  ));
```
Открываешь круглую скобку вместо фигурной, и js думает что у тебя функция в 1 строку. Тогда можно не писать return внутри. Чуть позже на реакте в основном только так и будем писать)

Reply:
Понял, подправлю :)